pyproject.toml

```diff
@@ -12,33 +12,28 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = "^3.10"
-beautifulsoup4 = "~4.9.3"
-boto3 = "~1.26.3"
-dj-database-url = "~1.0.0"
+python = "^3.11"
 django = "~3.2"
```

Comment:
Did you intend to upgrade Django as well?

Reply:
🤦‍♂️ yup. will take care of it in the afternoon

Reply:
👍 can always do it in the next upgrade cycle too. Django 3.2 is still LTS.